src/test/Components/CreateImageWizardV2/CreateImageWizard.test.tsx

```diff
@@ -952,6 +952,10 @@ describe('Step Upload to AWS', () => {
       // skip repositories
       await clickNext();
       // skip Details
+      const blueprintName = await screen.findByRole('textbox', {
+        name: /blueprint name/i,
+      });
+      await user.type(blueprintName, 'valid-name');
```

Comment:
this block is added because blueprint name is required, a bit long test file :sweat_smile:  later can be refactored to be more dry.